doc/docs.md

```diff
@@ -1082,6 +1082,7 @@ There are further built-in methods for arrays:
 * `a.delete_last()` removes the last element
 * `a.first()` equivalent to `a[0]`
 * `a.last()` equivalent to `a[a.len - 1]`
+* `a.single()` equivalent to `a[0]` but also panics if it contains more that one element
```

Comment:
This needs to be improved.  Right now, it says that if the array has more than one element, the routine will panic.

Maybe something like
```suggestion
* `a.single()` equivalent to `a[0]` but also panics if it the element is not unique
```